bittensor/core/extrinsics/async_weights.py

```diff
@@ -155,6 +223,88 @@ async def set_weights_extrinsic(
         return False, str(error)
 
 
+async def batch_set_weights_extrinsic(
+    subtensor: "AsyncSubtensor",
+    wallet: "Wallet",
+    netuids: list[int],
+    uidss: list[Union[NDArray[np.int64], "torch.LongTensor", list]],
+    weightss: list[Union[NDArray[np.float32], "torch.FloatTensor", list]],
```

Comment:
Typos? Also is there a reason we accept `torch.LongTensor` objects for `set_weights` but not for `commit_weights`?